pkg/printers/internalversion/describe.go

```diff
@@ -1264,6 +1264,20 @@ func describePersistentVolumeClaim(pvc *api.PersistentVolumeClaim, events *api.E
 		if pvc.Spec.VolumeMode != nil {
 			w.Write(LEVEL_0, "VolumeMode:\t%v\n", *pvc.Spec.VolumeMode)
 		}
+		if len(pvc.Status.Conditions) > 0 {
```

Comment:
We are doing something similar in `describeNode`, would you mind creating a helper function and place it there?

Reply:
I'm not sure which code we can extract as a helper function, `pvc.Status.Conditions`(its type is `[]PersistentVolumeClaimCondition`) has a different type from `node.Status.Conditions`(its type is `[]NodeCondition`). If we really want to do this, we can only extract the print operation(as follows) as a helper function, Is this what you want?

```go
w.Write(LEVEL_1, "%v \t%v \t%s \t%s \t%v \t%v\n",
	c.Type,
	c.Status,
	c.LastProbeTime.Time.Format(time.RFC1123Z),
	c.LastTransitionTime.Time.Format(time.RFC1123Z),
	c.Reason,
	c.Message)
```

Reply:
Ah, sorry my bad I didn't notice that.